test/integration/history_int_test.go

```diff
@@ -18,7 +19,15 @@ func (suite *HistoryIntegrationTestSuite) TestHistory_History() {
 	defer ts.Close()
 
 	ts.LoginAsPersistentUser()
-	cp := ts.Spawn("history", "--namespace", "ActiveState-CLI/History")
+
+  cp := ts.Spawn("checkout", "ActiveState-CLI/History")
+  cp.Expect("Checked out")
+  cp.ExpectExitCode(0)
```

Comment:
nit: Is this tabbing correct? Looks off to me.

Reply:
Good eye. It was off.